Java-Data-Types.md

```diff
@@ -0,0 +1,148 @@
+# Java data-Types
+
+Java is a strongly ­typed language. This means that, in Java, each data type has its own strict definition. There are no implicit data type conversions when any conflicts occur between the data types. Any change in data types should be explicitly declared by the programmer.
+ 
+Java defines 8 simple data types. They are divided into the following categories:
+ 
+Integers
+ 
+Floating­ Point Numbers
+ 
+Characters
+ 
+Boolean Type
+ 
+The details of each of the data types is given below
+ 
+### Integers:
+
+These are of four types: byte, short, int, long. It is important to note that these are signed positive and negative values. Java does not support positive­ only numbers.
+
+#### byte:
+
+    Byte data type is an 8-bit signed two's complement integer.
+
+    Minimum value: -128 (-2^7)
+
+    Maximum value: 127 (inclusive)(2^7 -1)
+
+    Default value: 0
+
+    Example: byte a = 10 , byte b = -500
+
+#### short:
+
+    Short data type is a 16-bit signed two's complement integer.
```

Comment:
See, this is what sets a good writer apart from a moderate writer. It's the amount of information you throw at people within a given amount of time or breadth of an article.

These articles should not put off beginners. You are using the term "two's complement integer" without any explanation as to what they mean.

At the very least, link to a Wikipedia article or some lucid article as to what they are. Not everyone would have an understanding of that.